src/linker/Linker/Driver.cs

```diff
@@ -218,6 +219,10 @@ public void Run (ILogger customLogger = null)
 							}
 							continue;
 
+						case "--annotate-method-access":
```

Comment:
This needs to be documented and please try to come up with more specific name (this is too vague)